src/schema/rules/sidecars/asl.yaml

```diff
@@ -31,7 +31,23 @@ MRIASLTextOnlySliceTiming:
     - suffix == "asl"
     - sidecar.MRAcquisitionType == "2D"
     fields:
-        SliceTiming: REQUIRED
+        SliceTiming:
+            level: REQUIRED
+            code:
+                SLICE_TIMING_NOT_DEFINED_2D_ASL: |
+                    "You should define SliceTiming', in case 'SequenceType' is set to a
```

Comment:
I realize it would be more verbose, but I kind of prefer something like
```
code:
    name: SLICE_TIMING_NOT_DEFINED_2D_ASL
    message: |
        ....
```

Reply:
Yeah, I was going to ask about that. My thoughts were just to collect the code keys and the message would follow (or be empty). I'm happy as long as the code name and message are contained w/ in the code key as there's nothing wrong with being verbose/explicit. @ericearl can you take a look at this and potentially update your code entries to what @tsalo suggested.